### PR TITLE
Fixes DOC-3041

### PR DESCRIPTION
**Pull request by lamagnifica**
Replaces https://github.com/edx/edx-documentation/pull/1106
## [DOC-3041](https://openedx.atlassian.net/browse/DOC-3041)

The XBlock URL has been available for some time. To add the URL info to multiple guides, I created a new "shared" file and then included it in both the XBlock tutorial and in the Developer's Guide. 

Reviewers, is the info about the URL complete and accurate? Have I included it in the right places in the XBlock Tutorial and the Developer's guide?
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @nasthagiri 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check/copy edit/dev edit): pdesjardins 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @nedbat I understand that this gets asked about on Slack & edx-code
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/dev/Section_XBlock_URL.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. _Section_XBlock_URL:
```
Comment (lamagnifica) [inline en_us/shared/dev/Section_XBlock_URL.rst:1]: @nasthagiri this is the important file in this PR. 
I expanded it to describe what a usage ID is, and added pointers to the pre-existing documentation for how to find or call it. I also added images for the video example to show the difference with/without Chrome

File: en_us/developers/source/extending_platform/xblocks.rst [hunk 2]
```diff
@@ -1,110 +1,142 @@
...
+  application. Ideally interesting state of the XBlock could be reconstructed
+  at any point in history through careful analysis of the event stream.
 
-  TODO: Link to the authoritive list of event types.
+..  TODO: Link to the authoritative list of event types.
 
-In the future, these are likely to become more formal XBlock services (one related to users,
-and the other to event publishing).
+In the future, these are likely to become more formal XBlock services (one
+related to users, and the other to event publishing).
```
Comment (nasthagiri) [inline en_us/developers/source/extending_platform/xblocks.rst:45]: By the way, we also have xBlock runtime services for Users and Internationalization.  Perhaps @cpennington or @ormsbee may have more information on whether they should be documented.

File: en_us/shared/dev/Section_XBlock_URL.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+To use the XBlock URL and return the HTML rendering of an individual XBlock,
+you use the following URL path for an XBlock on an edX site.
+
+``https://{host}/xblock/{usage_id}``
+
+========================
+Finding the ``usage_id``
+========================
+
+The ``usage_id`` is the unique identifier for the problem, video, text, or
```
Comment (nasthagiri) [inline en_us/shared/dev/Section_XBlock_URL.rst:19]: Nit: Can you add here that `usage_id`s are also identifiers for container course components such as `sequential`s and `vertical`s?   For example, the following URL presents the student view of the top-level sequential: https://courses.edx.org/xblock/block-v1:edX+VideoX+1T2016+type@sequential+block@1a52e689bcec4a9eb9b7da0bf16f682d

**Comment by nasthagiri** [comment]
👍   2 questions.  Otherwise, lgtm.

File: en_us/developers/source/extending_platform/xblocks.rst (hunk 2 above)
Comment (cpennington) [inline en_us/developers/source/extending_platform/xblocks.rst:45]: Yes, they probably should be.

Comment (lamagnifica) [inline en_us/developers/source/extending_platform/xblocks.rst:45]: @cpennington and @nasthagiri I leave it to you to create the DOC story or stories that are needed.

**Comment by lamagnifica** [comment]
error is for the new anchor -- tests fail until it is available in the HTML

**Pull request merged**